Realm/RLMRealm.mm

```diff
@@ -89,9 +89,6 @@ static bool shouldForciblyDisableEncryption() {
         if (key.length != 64) {
             @throw RLMException(@"Encryption key must be exactly 64 bytes long");
         }
-        if (RLMIsDebuggerAttached()) {
-            @throw RLMException(@"Cannot open an encrypted Realm with a debugger attached to the process");
-        }
 #if TARGET_OS_WATCH
         @throw RLMException(@"Cannot open an encrypted Realm on watchOS.");
 #endif
```

Comment:
I think the switch away from Mach exceptions should allow encrypted Realms to work on watchOS and tvOS now?